modular_skyrat/master_files/code/modules/awaymission/mission_code/black_mesa/mesa_mobs.dm

```diff
@@ -549,6 +549,8 @@
 	desc = "A 5.56mm surplus bullet casing."
 	projectile_type = /obj/projectile/bullet/a556/weak
 
+#define MOB_PLACER_RANGE 12 // One more tile than the biggest viewrange we have.
```

Comment:
CONFIG_GET(string/default_view) + 1 ?

Reply:
That relies on the config being set tho

Reply:
I don't see many instances where the config wouldn't be set.